content/post/28-sept-21-oss.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Welcome to OSDC Newsletter!"
+---
+
+<!-- Introduction to Newsletter -->
+
+We will be introducing y'all with the world of free and open source software this week.
+
+# Open Source Explainers
+
+# Open Source Motivation - First hand experiences
+
+# News 
+
+<!-- Add tags/metadata maybe -->
+
+# Conferences/Meetups
+
+# Major feat releases
+
+<!-- go generics sorta? -->
+
+# Blogposts by the community members
+
+# Tip of the week
```

Comment:
Tip/Tool of the week, drop in your suggestions we will choose the best one or most voted one. ( Thumbs up if you like someone's team )